packages/sitecore-jss-cli/src/scripts/deploy.files.ts

```diff
@@ -65,6 +66,18 @@ export const builder = {
 export async function handler(argv: any) {
   verifySetup();
 
+  const packageJson = await resolvePackage();
+
+  if (packageJson.config.serverSideRenderingEngine) {
```

Comment:
As opposed to a new config property, couldn't we use the presence of the existing `sitecoreDistPath`? Next.js sample app is the only one that won't have this.
```suggestion
  if (!packageJson.config.sitecoreDistPath) {
```

Reply:
@ambrauer Good idea. I didn't notice it. Added